Tools/jit/template.c

```diff
@@ -43,6 +43,7 @@
 #undef GOTO_TIER_TWO
 #define GOTO_TIER_TWO(EXECUTOR) \
 do {  \
+    OPT_STAT_INC(traces_executed);                \
```

Comment:
I think this only catches tier-two-to-tier-two transfers, but not tier-one-to-tier-two transfers. We should probably also add this to the JIT definition of `GOTO_TIER_TWO` in `ceval_macros.h` so it's picked up by tier one.

Reply:
At that point, we should probably move the existing counter for this from `ceval.c` to the non-JIT definition of `GOTO_TIER_TWO` in `ceval_macros.h` so we're counting it in the same place on both builds (and there's less chance that a refactoring would break one).

Reply:
That makes sense.